src/lsst/rsp/service.py

```diff
@@ -0,0 +1,54 @@
+import os
+
+from pyvo.dal import SIA2Service
+from pyvo.dal.adhoc import DatalinkResults
+from pyvo.dal.sia2 import ObsCoreRecord
+
+from .utils import get_pyvo_auth, get_service_url
+
+LSST_CLOUD = [
```

Comment:
This feels like something we should drive from a configmap that we mount into a pod rather than embedding directly.  That might be a future enhancement though.

Reply:
Yeah, right now this code doesn't even do all that much.  But I wasn't ready to make a config map and go through all the work of the jupyter stuff, but probably when we actually have 2 services running I'll consider it.  I'm not even sure these are the rules Frossie wants.